test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/SignCommandTests.cs

```diff
@@ -707,7 +707,7 @@ public async Task SignCommand_SignPackageWithUnsuportedTimestampHashAlgorithm_Sh
 
                     Assert.False(result.Success);
                     Assert.Contains(_timestampUnsupportedHashAlgorithmCode, result.AllOutput);
```

Comment:
Rename this to `_timestampUnsupportedDigestAlgorithmCode`

Reply:
Thanks! Fixed.